tests/experiment/test_base_experiment.py

```diff
@@ -349,7 +349,7 @@ def test_set_reset_true_with_connected_device(
         exp.platform.connect()
         exp.platform.disconnect()
         mock_reset.assert_called()
-        assert mock_reset.call_count == 14
+        assert mock_reset.call_count == 12
```

Comment:
What is the reason for this change? Has anything changed so that this test method needs to be readjusted?